<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GearDialogDesign</class>
 <widget class="QWidget" name="GearDialogDesign">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="10" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="chemin">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="titre">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="22" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QDoubleSpinBox" name="volume">
       <property name="suffix">
        <string> %</string>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="value">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true"/>
     </item>
    </layout>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Notes</string>
     </property>
    </widget>
   </item>
   <item row="31" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Titre*</string>
     </property>
    </widget>
   </item>
   <item row="17" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Fondue Debut</string>
     </property>
    </widget>
   </item>
   <item row="27" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Lire Suivant</string>
     </property>
    </widget>
   </item>
   <item row="21" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Fondue Fin</string>
     </property>
    </widget>
   </item>
   <item row="22" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Volume</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_5">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Chemin*</string>
     </property>
    </widget>
   </item>
   <item row="25" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Lire en Boucle</string>
     </property>
    </widget>
   </item>
   <item row="25" column="1">
    <widget class="QCheckBox" name="boucle">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="31" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>* Champs Obligatoires</string>
     </property>
    </widget>
   </item>
   <item row="21" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QDoubleSpinBox" name="fondueStop">
       <property name="suffix">
        <string> s</string>
       </property>
       <property name="value">
        <double>5.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="fondueStopCheck">
       <property name="text">
        <string>Actif</string>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="17" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QDoubleSpinBox" name="fondueStart">
       <property name="suffix">
        <string> s</string>
       </property>
       <property name="value">
        <double>5.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="fondueStartCheck">
       <property name="text">
        <string>Actif</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="12" column="1">
    <widget class="QTextEdit" name="notes">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="acceptRichText">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="32" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QPushButton" name="del">
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="Reset">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="edit">
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="add">
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Debut</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Fin</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="pageListe"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>15</height>
      </size>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="29" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Position*</string>
     </property>
    </widget>
   </item>
   <item row="27" column="1">
    <widget class="QCheckBox" name="next">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QTimeEdit" name="fin">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="accelerated">
        <bool>true</bool>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>true</bool>
       </property>
       <property name="displayFormat">
        <string>HH:mm:ss:zzz</string>
       </property>
       <property name="timeSpec">
        <enum>Qt::LocalTime</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_3" native="true"/>
     </item>
    </layout>
   </item>
   <item row="13" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QTimeEdit" name="debut">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="accelerated">
        <bool>true</bool>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>true</bool>
       </property>
       <property name="displayFormat">
        <string>HH:mm:ss:zzz</string>
       </property>
       <property name="timeSpec">
        <enum>Qt::LocalTime</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_4" native="true"/>
     </item>
    </layout>
   </item>
   <item row="28" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Onglet*</string>
     </property>
    </widget>
   </item>
   <item row="28" column="1">
    <widget class="QComboBox" name="pageListePos"/>
   </item>
   <item row="29" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QComboBox" name="apres">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Apr√®s</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Avant</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="gearListePos"/>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="gearListe"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Controleur</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>Onglet</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>titre</tabstop>
  <tabstop>chemin</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>boucle</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
